components/org.wso2.carbon.identity.api.server.secret.management/org.wso2.carbon.identity.api.server.secret.management.common/src/main/java/org/wso2/carbon/identity/api/server/secret/management/common/Utils.java

```diff
@@ -18,7 +18,7 @@
 
 package org.wso2.carbon.identity.api.server.secret.management.common;
 
-import org.apache.log4j.MDC;
+import org.apache.logging.log4j.ThreadContext;
```

Comment:
can't we use org.slf4j.MDC here?

Reply:
Yes, Later this PR is merged with the improvement. https://github.com/wso2/identity-api-server/pull/360